src/primitives/redpallas.rs

```diff
@@ -18,7 +18,7 @@ impl SigType for Binding {}
 
 /// A RedPallas signing key.
 #[derive(Debug)]
-pub struct SigningKey<T: SigType>(reddsa::SigningKey<T>);
+pub struct SigningKey<T: SigType>(pub(crate) reddsa::SigningKey<T>);
```

Comment:
What is this being exposed to the crate for? Nothing outside of this module should be interacting with RedDSA types directly (and they will be completely inaccessible once there is a `redpallas` crate).